redi-connect-front/src/components/organisms/SideMenu.tsx

```diff
@@ -27,15 +41,15 @@ const SideMenu = () => {
 
   return (
     <ul className="side-menu">
-      <MenuItem url="/app/me">My Profile</MenuItem>
+      <MenuItem url="/app/me" icon="profile">My Profile</MenuItem>
       {(profile.userType === 'mentee' || profile.userType === 'public-sign-up-mentee-pending-review') &&
-        <MenuItem url="/app/dashboard">Find a mentor</MenuItem>
+        <MenuItem url="/app/dashboard" icon="mentorship">Find a mentor</MenuItem>
```

Comment:
For simplicity reason why not add the icon directly as a child? That way you can save a prop and an if too.

Reply:
Good idea, thanks!